src/locales/cn.json

```diff
@@ -40,59 +40,59 @@
   },
 
   "home":{
-    "meta-title": "ScreenToGif - Record your screen, edit and save as a gif, video or other formats",
-    "meta-description": "Free screen recorder tool, which lets you record, edit and save as a gif, video or other formats.",
-
-    "subtitle": "<strong>Screen</strong>, <b>webcam</b> and <b>sketchboard</b> recorder with an integrated <b>editor</b>",
-    "version": "Version {0}",
-    "installer" : "Installer",
-    "installer-info": "Downloads the installer version (.msi), which contains the main executable and optional addons.",
-    "or": "or",
-    "portable" : "Portable",
-    "portable-info": "Downloads the portable version, which contains only the main executable. Addons needs to be downloaded in Options > Extras.",
-    "downloads" : "{0} downloads",
-    "more-releases" : "More download options",
-    "fewer-releases" : "Fewer download options",
-    "copy" : "Copy",
-    "copied" : "Command copied!",
-    "not-copied" : "It was not possible to copy the command.",
+    "meta-title": "ScreenToGif - 录屏，编辑，保存为GIF，视频或更多其他格式",
+    "meta-description": "免费录屏工具，帮你完成录屏，编辑，保存为GIF，视频或更多其他格式。",
+
+    "subtitle": "<strong>屏幕</strong>，<b>摄像头</b>和<b>画板</b>录像，并有内置<b>编辑器</b>",
+    "version": "版本 {0}",
+    "installer" : "安装包",
+    "installer-info": "下载安装包 (.msi)，其中包含主执行程序和可选插件。",
+    "or": "或",
+    "portable" : "免安装包",
+    "portable-info": "下载免安装包，包含主执行程序。插件可从 Options > Extras 下载。",
```

Comment:
I just think that Options > Extras should be translated too, since it's displayed in Chinese inside the app too.

![image](https://user-images.githubusercontent.com/14798947/93278383-dfcd0100-f79a-11ea-810c-f5bc3628ed5d.png)

Reply:
Yes it should. I just didn't do it because I haven't got to translate "options" and "extras". I wanted to keep them in sync so there are no confusions.

Reply:
Looks like the translation is mostly done already?